concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1408,6 +1415,13 @@ jobs:
                 credhub set --name=/concourse/main/create-cloudfoundry/logit-elasticsearch-api-key --type password --password "${LOGIT_ELASTICSEARCH_API_KEY}"
```

Comment:
Can we encapsulate the namespace in a variable? That'd make it easier to maintain if we needed to change the namespace in the future.

```
credhub set --name=${CREDHUB_NS}/logit-elasticsearch-api-key --type password ${LOGIT_ELASTIC_SEARCH_API_KEY}
```
```